doc/manual/de/config/widgets/plugins/tr064/index.rst

```diff
@@ -128,6 +125,7 @@ Hier der minimale Beispielcode der das calllist Widget aus dem tr064 Plugin aus
     </meta>
     <calllist device="fritzbox">
         <label>calllist</label>
+        <address transform="DPT:1.001" mode="read">1/1/0</address>
```

Comment:
Is the `<address>` needed by the new functionality?
The tr064 plugin/calllist widget should work without it.

Reply:
Well then the schema definition is wrong:
```
<xsd:complexType name="calllist">
    <xsd:sequence>
      <xsd:element name="layout" type="layout" minOccurs="0" maxOccurs="1"/>
      <xsd:element name="label" type="label" minOccurs="0" maxOccurs="1"/>
      <xsd:element name="address" type="address" minOccurs="1" maxOccurs="unbounded"/>
    </xsd:sequence>
```
The Screenshot tool validates the config snippet and those are currently invalid without an address. Should I change the schema in this PR or do you want to create one (which should be merged before this one then.

Reply:
You can change the XSD with this PR (line 1870 to `minOccurs="0"` I guess)